lib/nrf91_lwm2m_app/Kconfig

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF91_LWM2M_APP
+	bool "nRF91 LWM2M client application library"
+	select LWM2M
+
+if NRF91_LWM2M_APP
+
+config NRF91_LWM2M_APP_CONN_MON_OBJ_SUPPORT
+	bool "Connection Monitor object support"
+	default y
+	select LWM2M_CONN_MON_OBJ_SUPPORT
+
+if NRF91_LWM2M_APP_CONN_MON_OBJ_SUPPORT
+
+config LWM2M_CONN_MON_BEARER_MAX
+	int "Max number of bearers"
+	default 2
+
+endif # NRF91_LWM2M_APP_CONN_MON_OBJ_SUPPORT
+
+config NRF91_LWM2M_APP_LOCATION_OBJ_SUPPORT
+	bool "Location object support"
+	default y
+	select LWM2M_CONN_LOCATION_OBJ_SUPPORT
+
+config NRF91_LWM2M_APP_FIRMWARE_UPDATE_OBJ_SUPPORT
```

Comment:
@rlubos Are these macros something what you were looking for?

Reply:
yes, thank you